iOS-Weather-App/Controllers/CurrentDetailedWeatherViewController/CurrentDetailedWeatherViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Created by pavels.garklavs on 23/02/2022.
+//
+
+import UIKit
+
+class CurrentDetailedWeatherViewController: UIViewController {
+
+    private let detailsTableView = UITableView()
```

Comment:
you haven't got others tableView at this class, think that you can rename this property to simple tableView